### PR TITLE
[docs] simplify run a fullnode steps

### PR DESCRIPTION
**Pull request by geekflyer**
### Description

this simplifies the steps to run a fullnode a bit

### Test Plan
ran through it myself and updated the docs as needed

### CODE REVIEW

**Review by rajkaramchedu — APPROVED** [review]

File: developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md [hunk 1]
```diff
@@ -97,7 +97,8 @@ With your development environment ready, now you can start to setup your FullNod
 
 5. Checkout the `devnet` branch using `git checkout --track origin/devnet`.
 
-6. Make sure your current working directory is `aptos-core`. Copy the YAML configuration file from `config/src/config/test_data/public_full_node.yaml` to your current working directory. You will edit this file to ensure that your FullNode:
+6. Make sure your current working directory is `aptos-core`. 
+   Run `cp config/src/config/test_data/public_full_node.yaml fullnode.yaml` to create a copy of the fullnode config template. You will edit this file to ensure that your FullNode:
```
Comment (sherry-x) [inline developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md:101]: Could you please also update the upgrade instruction for this change? https://aptos.dev/tutorials/full-node/update-fullnode-with-new-releases

**Review by sherry-x — APPROVED** [review]
Overall looks great! Thanks for the improvements! Just one small thing to also update the file names referred in the upgrade document

**Comment by geekflyer** [comment]
ready for re-review! :)

File: developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md (hunk 1 above)
Comment (geekflyer) [reply developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md:101]: done

**Pull request closed without merging**